src/components/Header.test.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from "@testing-library/react";
+import Header from "./Header";
+
+describe("<Header/>", () => {
```

Comment:
Minor... estamos usando este enfoque cuando queremos hacer referencia al componente como en este caso, me agrada porque si renombramos el componente podremos ver que test describe debemos modificar sin esfuerzo e incluso puede que automáticamente se actualicen como es el caso en _Webstrom IDE_
```suggestion
describe(Header.name, () => {
```

Reply:
Me parece buena idea @Cromeror 
Consulta: al agregar este enfoque en EditorWebApp, debieron desactivar el chequeo de eslint _Title must be a string_ ? `ignoreTypeOfDescribeName`